### ISSUE TITLE
1.0 - 1.4.6 support

### ISSUE THREAD

**Issue by candrawardana** [ProtocolSupport/ProtocolSupport#1312]
please add 1.0 to 1.4.6 support, i know it's buggy

**Comment by WolfyRed** [comment]
Hmm we need the Protocol API first, or something similar

**Comment by Shevchik** [comment]
Maybe in the far far future when i have literally nothing else to do

**Comment by LegacyGamerHD** [comment]
This world be nice

**Comment by kangarko** [comment]
Would be awesome to see at least 1.2.5 support :)

**Comment by FlorianMichael** [comment]
Classic support when

**Comment by Logics4** [comment]
Plz add support for Cave Game /s

**Comment by FlorianMichael** [comment]
java to bedrock support when

**Comment by VytskaLT** [comment]
Why would anyone want to use these ancient versions?

**Comment by FlorianMichael** [comment]
cause they are cool

**Comment by kangarko** [comment]
Because we grew up on them and I am getting old :P

**Comment by xIcee** [comment]
add back the aPril foosl udate!11!!!1!

**Comment by kangarko** [comment]
Nah, 1.2.5 compatibility is fine haha, I got my plugin ChatControl to work on that rofl

**Comment by xIcee** [comment]
I can't believe this is still open. So this is still completely on the table? That's crazy

**Comment by poopman97** [comment]
pre-classic support when

**Comment by xIcee** [comment]
add enigma machine support?
![Screenshot from 2022-09-07 01-20-34](https://user-images.githubusercontent.com/86935554/188794906-ad398cf1-e89d-48d1-adc9-89038d538908.png)

**Comment by artdeell** [comment]
> add enigma machine support? ![Screenshot from 2022-09-07 01-20-34](https://user-images.githubusercontent.com/86935554/188794906-ad398cf1-e89d-48d1-adc9-89038d538908.png)

you'd need to make java run on it first

**Comment by itsTyrion** [comment]
![Nagajari-khanikargaon-rock-inscription](https://github.com/ProtocolSupport/ProtocolSupport/assets/17730120/d372c39e-254b-47a3-86a2-f586de1fdb5e)
When is support for this coming??

**Comment by Outfit8TSB** [comment]
I Will Do It On My Project
See The Progress on https://github.com/GAME-CLI-SRV-DEV/ViaVersionProtocolSupport
And for this time i will go to Classic 0.0.15!!

**Comment by Outfit8TSB** [comment]
> java to bedrock support when 

Going to do it on VVPS-Nukkit

**Comment by Outfit8TSB** [comment]
> Classic support when

Gonna do it on VVPS

**Comment by kangarko** [comment]
> I Will Do It On My Project See The Progress on https://github.com/GAME-CLI-SRV-DEV/ViaVersionProtocolSupport And for this time i will go to Classic 0.0.15!!
> 
> Update: when i started the work on 1.6.4 Client -> 1.7.2~5 Client. ViaVersion Does not Recognize the Translation i made and says that "Packet LOGIN in r1_7_2_5tor1_6_4 has no mapping. It Must Be Remapped or Cancelled." What Does That Even Mean? I Remapped it! The Protocol File "No Mapping" was Occurred: src/main/java/me/falixsrv/approximasteranarchy2004/ViaVersionProtocolSupport/Protocol/r1_7_2_5tor1_6_4/r1_7_2_5tor1_6_4.java
> 
> Can @FlorianMichael Solve it?

Good luck with that! Just don't give up because it could be an amazing project. Unfortunately most devs disappear after a while with these ideas. Hope to see you in the next year too!

**Comment by Outfit8TSB** [comment]
Hope to see you in a next year too
